src/App.tsx

```diff
@@ -1,26 +1,97 @@
-/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList/TodoList';
+import { FilterTypes } from './types/FilterTypes';
+import { getPreparedTodos } from './utils/GetPrepatedTodos';
+import { getAmountOfActiveTodos } from './utils/getAmountOfActiveTodos';
+import { Footer } from './components/Footer/Footer';
+import { Errors } from './utils/Errors';
+import { ErrorsField } from './components/ErrorsField/ErrorsField';
+import { Header } from './components/Header/Header';
+import { USER_ID } from './utils/CONSTANTS';
+import { handleError } from './utils/handleError';
+import { deletingTodo } from './utils/deletingTodo';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterInstructions, setFilterInstructions] = useState<FilterTypes>(
+    FilterTypes.All,
+  );
+  const [errorMessage, setErrorMessage] = useState<Errors>(Errors.Default);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+
+  useEffect(() => {
+    getTodos()
+      .then(setTodos)
+      .catch(() => {
+        handleError(Errors.Loading, setErrorMessage);
+      });
```

Comment:
It's better to move the logic inside the useEffect hook to a separate function, as it improves code readability, reusability, and makes testing easier. It also helps keep useEffect focused on side effects.